banking-app/demo_governance.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -euox pipefail
```

Comment:
The x in this comment prints out every command in the shell - I don't know the answer but do we want to do this for the governance calls? Maybe remove the x on this one

Reply:
How to use the governance mechanism is an important part of the demo. So I think it's worth to show.
Otherwise we can just use `--initial-member-count` option of sandbox.sh.

I am not sure if we should show every tiny detail like copying vote response file though.